classes/class-apm-admin-settings.php

```diff
@@ -285,9 +315,10 @@ public static function save() {
 						}
 						if ( 'CP' == $conditions[ $i ] ) {
 							$sub_key = array_search( $i, $cp_keys );
+							$ss_cp_condition = isset( $_POST['ss_cp_condition'][ $sub_key ] ) ? $_POST['ss_cp_condition'][ $sub_key ] : '';
```

Comment:
`sanitize_text_field` can be called for the `$_POST['ss_cp_condition'][ $sub_key ]` instead of separate condition below.